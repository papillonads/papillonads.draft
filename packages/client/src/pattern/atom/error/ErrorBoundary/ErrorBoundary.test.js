import errorBoundaryDefault from './ErrorBoundary.render'

describe('<ErrorBoundary />', () => {
  describe('Snaphot', () => {
    test('must match errorBoundaryDefault', () => {
      expect(global.renderToJSON(errorBoundaryDefault)).toMatchSnapshot()
    })
  })
})
