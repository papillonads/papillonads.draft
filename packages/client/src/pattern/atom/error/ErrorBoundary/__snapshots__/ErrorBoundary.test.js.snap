// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorBoundary /> Snaphot must match errorBoundaryDefault 1`] = `<div />`;
